src/Passable.js

```diff
@@ -31,7 +31,7 @@ class Passable {
         return this.res;
```

Comment:
at the top of this file you have `const FAIL: Severity = 'fail';` you are not exporting this const and it seems nobody use it in the file, maybe you can remove it?